config/areas/site/buttons.php

```diff
@@ -17,7 +17,7 @@
 		);
 	},
 	'page.preview' => function (Page $page) {
-		if ($page->permissions()->can('preview') === true) {
+		if ($page->permissions()->preview() === true) {
```

Comment:
When working on the user roles, I ran into some problems where using the magic caller methods was inferior to directly calling the `->can()` method. And tried to start moving away from using the magic methods. I can't remember right now what was the exact problem (maybe related to testing) - I'll try to remember.